components/brave_webtorrent/extension/background/webtorrent.ts

```diff
@@ -30,7 +30,7 @@ export const createServer = (torrent: WebTorrent.Torrent, cb: (serverURL: string
     origin: window.location.origin,
     // Use hostname option to mitigate DNS rebinding
     // Ref: https://github.com/brave/browser-laptop/issues/12616
-    hostname: 'localhost'
+    hostname: '127.0.0.1'
```

Comment:
would this break the check in https://github.com/webtorrent/webtorrent/pull/1678/files#diff-c945a46d13b34fcaff544d966cffcabaR76? i thought `req.headers.host` was `localhost` for these requests but don't really remember

Reply:
sorry just saw your comment; i guess it was broken before

Reply:
It's the other way around. 😂 The previous check-in broke everything because `req.headers.host` is actually `127.0.0.1`. I thought it would be easiest to just merge the previous PR to fix the security issue and then fix the breakage in this PR.